Makefile.include

```diff
@@ -55,15 +54,13 @@ __DIRECTORY_VARIABLES := \
   BUILD_DIR \
   BINDIRBASE \
   BINDIR \
-  CCACHE_BASEDIR \
   GITCACHE \
   PKGDIRBASE \
   DLCACHE_DIR \
   #
 
 # Make all paths absolute.
 override RIOTBASE       := $(abspath $(RIOTBASE))
-override CCACHE_BASEDIR := $(abspath $(CCACHE_BASEDIR))
```

Comment:
this is essential for CI, unsetting this multiplies CI cache usage, causing 50-100% slowdown.

Reply:
Isn't the correct `CCACHE_BASEDIR` for murdock set here?
https://github.com/RIOT-OS/RIOT/blob/master/.murdock#L234

Reply:
Ah, that's correct, murdock shouldn't be affected.